core/Plugin/Manager.php

```diff
@@ -447,7 +447,7 @@ public static function getPluginDirectory($pluginName)
             return self::$pluginsToPathCache[$pluginName];
         }
 
-        $corePluginsDir = PIWIK_INCLUDE_PATH . 'plugins/' . $pluginName;
+        $corePluginsDir = PIWIK_INCLUDE_PATH . '/plugins/' . $pluginName;
```

Comment:
maybe we to a `rtrim(PIWIK_INCLUDE_PATH, '/')` just in case? I think tests add for example a slash at the end to the constant but not sure.

Reply:
actually there is currently only one place, where we do a trim on `PIWIK_INCLUDE_PATH`:

https://github.com/matomo-org/matomo/blob/6395855aa7f3813cc7f413a18d31505ea26ba32a/core/Plugin/Manager.php#L477

But there are so many more uses so, I don't think it's actually required.

Reply:
:+1: looks good to merge